contracts/TokenVesting.sol

```diff
@@ -35,22 +43,36 @@ contract TokenVesting is Ownable {
     totalPeriods = _totalPeriods;
     timePerPeriod = _timePerPeriod;
   }
+  
+   /**
+   *@dev grapper for gurrent time to make contract testable
```

Comment:
@adamskrodzki can you rmeove this function? You dont need it for testing.

Reply:
Yes I need Id I use it, so in TokenVestingMock
i can override it and by that manipulate time

Using increaseTimeTo

require using also snapshots otherwise time in ganache-cli will not reverse and it will be hard to perform tests. 

This way code is much simplier.

Reply:
Here is another example of such approach

https://medium.com/level-k/testing-smart-contracts-with-truffle-7849b3d9961

Reply:
I ned a mock to set time (manipulate it) and therefore it need to be accessed by getter not directly. 

That is common way of testing code. To override pieces we want to manipulate.